satisfactory/chat.py

```diff
@@ -141,4 +163,36 @@ async def send_random_quote(self, context: CallbackContext) -> None:
     async def check_videos(self, context: CallbackContext) -> None:
         videos = self.youtube_client.get_new_videos()
         for video in videos:
-            await context.bot.send_message(chat_id=self.chat_id, text=video, parse_mode=ParseMode.HTML, disable_notification=True)
+            await context.bot.send_message(chat_id=self.chat_id, text=video, parse_mode=ParseMode.HTML,
+                                           disable_notification=True)
+
+    @Summary("satisfactory_bot_check_server_status", "Time spent running check_server_status").time()
+    async def check_server_status(self, context: CallbackContext) -> None:
+        if self.gameserver_address is None or self.gameserver_token is None:
+            return
+
+        try:
+            from pyfactorybridge import API
+            satisfactory_api_client = API(address=self.gameserver_address, token=self.gameserver_token)
```

Comment:
Should we move this to the `__init__` function so we only create a new API client instance once instead of every 10 seconds?

For the other API clients I even created the API clients outside in `cli.py` and passed the clients instead of the credentials. Though, I am not entirely sure if there was a reason.

Reply:
A refactoring that extracts the api client usage would surely be useful in case the functionality is expanded.

I intentionally did **not** move it into the init for now, because the API init code already runs multiple requests. If any of those requests fail there will be an exception and I didn't really know how to handle that smoothly yet. The current implementation simply relies on the try/catch and (as a last resort) the timer api. This is probably also something that can be implemented in a more sophisticated way once we extract the api client code to its own class (with appropriate and transparent error handling).

Reply:
I see, that is a fair point 👍 Then let us keep it like this for now :)